src/App.js

```diff
@@ -1,71 +1,86 @@
+import 'bootstrap/dist/css/bootstrap.min.css';
 import './App.css';
+import React from 'react';
 import axios from 'axios';
-import { Component } from 'react';
-import Figure from 'react'; 
-import Container from 'react';
+import Figure from 'react-bootstrap/Figure'; 
+import Container from 'react-bootstrap/Container';
+import Button from 'react-bootstrap/Button';
+import Form from 'react-bootstrap/Form';
 
-class App extends Component {
+class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
       searchQuery: '',
       location: {},
+      map: null,
       error: false,
     }
   }
-  handleChange = event => {
-    this.setState({
-      searchQuery: event.target.value,
-    });
-  }
-
-  getLocation = async () => {
+  // handleChange = event => {
+  //   this.setState({
+  //     searchQuery: event.target.value,
+  //   });
+  // }
 
+  getLocation = async (event) => {
+    event.preventDefault();
+    try {
     const url = `https://us1.locationiq.com/v1/search.php?key=${process.env.REACT_APP_CITY_KEY}&Q=${this.state.searchQuery}&format=json`;
 
-    try {
+    console.log(url);
       const response = await axios.get(url);
 
       const location = response.data[0];
-
+      console.log(location);
       this.setState({
         location: location,
         error: false,
       });
 
     } catch (error) {
-      console.error('Invalid City', this.state.searchQuery);
+      // console.error('Invalid City', this.state.searchQuery);
       this.setState ({ error: true});
     }
+    const map = `https://maps.locationiq.com/v3/staticmap?key=${process.env.REACT_APP_CITY_KEY}&center=${this.state.location.lat},${this.state.location.lon}&zoom=10`;
+
+    this.setState({
+      map: map,
+    })
   }
 
   render() {
     return (
       <Container>
-        <input onChange={(event) => this.setState({ searchQuery: event.target.value})} placeholder="enter city name"></input>
-        <button onClick={this.getLocation}>Explore!</button>
+        <>
+        <Form>
+        <Form.Label>Title</Form.Label>
+
+        <Form.Control onChange={(event) => this.setState({ searchQuery: event.target.value})} placeholder="enter city name" type="text"></Form.Control>
```

Comment:
probably dont need the fragments. I'm beginning to wonder if you need this.setState on line 59, but I would look to make sure that it isn't causing it to not render like intended.